test/stablex/stablecoin_converter.js

```diff
@@ -467,6 +466,62 @@ contract("StablecoinConverter", async (accounts) => {
       assert.equal((await stablecoinConverter.getBalance.call(user_2, erc20_2)).toString(), basicTrade.deposits[1].amount.sub(getExecutedSellAmount(fullBuyVolumes[1], prices[0], prices[1])).toString(), "Sold tokens were not adjusted correctly")
       assert.equal((await stablecoinConverter.getBalance.call(user_2, feeToken)), fullBuyVolumes[1].toString(), "Bought tokens were not adjusted correctly")
     })
+    it("Ensure fee token is burnt as expected", async () => {
+      // Fee token can't be a Mock here.
```

Comment:
Why not? It should be able to be mocked using `givenAnyReturn(true)`. And later we can inspect how it was called using https://github.com/gnosis/mock-contract/#inspect-invocations

Reply:
I was under the impression that I would need to verify the balances of this token and would, thus, need to deploy an actual token that could actually return valid information.